lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -2016,33 +2114,32 @@
     ],
     "incomplete": [
       {
```

Comment:
dunno why `color-contrast` went bye-bye

Reply:
> dunno why `color-contrast` went bye-bye

hmm, didn't this change once before?

Reply:
oh i guess it did and we decided to manually keep it around?